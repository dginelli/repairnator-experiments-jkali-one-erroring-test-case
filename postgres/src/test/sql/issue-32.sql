SELECT * FROM zdb_tally('so_posts', 'tags', '^.*', '', 12, 'count');