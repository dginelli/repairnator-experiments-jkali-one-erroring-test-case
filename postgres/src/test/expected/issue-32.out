SELECT * FROM zdb_tally('so_posts', 'tags', '^.*', '', 12, 'count');
    term     | count  
-------------+--------
             | 129680
 design      |   4955
 c           |   4283
 programming |   2842
 java        |   2713
 development |   2618
 web         |   2207
 patterns    |   1873
 object      |   1803
 oriented    |   1798
 code        |   1416
 testing     |   1393
 php         |   1344
(13 rows)

